hatchet_sdk/v2/callable.py

```diff
@@ -161,21 +172,13 @@ def get_action_name(self) -> str:
         return self.function_namespace + ":" + self.function_name
 
 
-T = TypeVar("T")
-
-
-class TriggerOptions(TypedDict):
-    additional_metadata: Dict[str, str] | None = None
```

Comment:
This is a dupe of `ChildTriggerWorkflowOptions`, so better to reuse that one IMO